keyboards/maartenwut/minitomic/readme.md

```diff
@@ -0,0 +1,23 @@
+# Minitomic
+
+![Minitomic](https://i.imgur.com/wGtNokW.jpg)
+
+## Support
+Keyboard Maintainer: [Maarten Dekkers](https://github.com/maartenwut)  
```

Comment:
We actually recently changed the template for this section. It's an unordered list now.

```suggestion
* Keyboard Maintainer: [Maarten Dekkers](https://github.com/maartenwut)
```